go/plumbing/example_test.go

```diff
@@ -0,0 +1,45 @@
+package plumbing
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+
+	"github.com/go-openapi/runtime"
+	httptransport "github.com/go-openapi/runtime/client"
+	"github.com/go-openapi/strfmt"
+	"github.com/netlify/open-api/go/plumbing/operations"
+)
+
+func Example() {
+	server := httptest.NewServer(http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
+		rw.Header().Set("Content-Type", "application/json; charset=utf-8")
+		rw.Write([]byte(`{ "name": "Pets of Netlify" }`))
+	}))
+	defer server.Close()
+	host := strings.ReplaceAll(server.URL, "http://", "")
+
+	// Create the API client
+	// For Netlify's production API use DefaultHost, DefaultBasePath, DefaultSchemes
+	transport := httptransport.New(host, DefaultBasePath, []string{"http"})
+	client := New(transport, strfmt.Default)
+
+	// Prepare the API token
+	authInfo := runtime.ClientAuthInfoWriterFunc(func(r runtime.ClientRequest, _ strfmt.Registry) error {
+		r.SetHeaderParam("User-Agent", "Your app")
+		r.SetHeaderParam("Authorization", "Bearer your_netlify_api_token")
+		return nil
+	})
```

Comment:
The bearer token can be set more cleanly like below, but I think it's a good idea to encourage API users to set the `User-Agent`.
```
authInfo := httptransport.BearerToken("your_netlify_api_token"))`
```